app/controllers/api/vcf_files_controller.rb

```diff
@@ -57,6 +58,8 @@ def create
       end
     end
     vcf.save
+    # QC check
+    passed, output = SimilarityJob.new(f).run
```

Comment:
move QC after the block of "unless p_services.empty?". If the pedia service is still running, we don't accept any submission from the same case. Therefore, we should put SimilarityJob after checking existed pedia service.